lib/register.js

```diff
@@ -1,24 +1,21 @@
 const parseUri = require('drachtio-srf').parseUri;
-const {isUacBehindNat, isWSS} = require('./utils');
+const {isUacBehindNat, getSipProtocol} = require('./utils');
 const NAT_EXPIRES = 30;
 
 module.exports = handler;
 
 function handler({logger}) {
   return (req, res) => {
     logger.info(`received ${req.method} from ${req.protocol}/${req.source_address}:${req.source_port}`);
-	
-    const fromUri = req.getParsedHeader('From').uri;
-    const aor = fromUri.match(/^sip:(.*)$/)[1];
-
-    req.registration = {
-	  aor: aor,
-	  expires: req.msg.headers.expires,
-	  contact: req.getParsedHeader('Contact')[0].uri,
-	  protocol: isWSS(req) ? 'wss' : req.protocol,
-	}
-
-    logger.info(req.registration, 'registration aor');
+    logger.info(req.registration, 'registration details');	
+    
+	try {
```

Comment:
per my comment below, now the calls to `register` or `unregister` should never throw, so need for try-catch